app/routes/events/EventDetailRoute.js

```diff
@@ -96,7 +102,8 @@ const mapStateToProps = (state, props) => {
     eventId,
     pools,
     registrations,
-    currentRegistration
+    currentRegistration,
+    currentRegistrationIndex
```

Comment:
This is missing in the Props Flow validation. 
`currentRegitrationIndex: number` needs to be added